docs/get-started/koios.md

```diff
@@ -0,0 +1,96 @@
+
+Let's start with basics from consumer point of view, and then we can dive to a brief summary about running a Koios Instance as a provider (independently as well as stringing in to existing instances).
+
+## Usage
+
+### API Documentation
+
+You can access API documentation [here](https://api.koios.rest). Koios leverages [PostgREST](https://postgrest.org/) to provide data, which means you can easily filter your data vertically as well as horizontally, and leverage in-built ordering/custom paging benefits as per the guide [here](https://api.koios.rest/#overview--api-usage). There isnt any centralised registry to be able to access the endpoints for most of the usage.
+
+Each endpoint on the document provides a sample curl command that can be used to test using an example, which can be executed directly from the browser for testing as well.
+
+![img](../../static/img/get-started/koios/1-usage.png)
+ 
+### Limits
+
+If you're using Koios service remotely, there are certain measures to protection against spamming or potentially unknowingly cause a Denial-of-Service against instance providers. Be sure to be mindful of the limits mentioned [here](https://api.koios.rest/#overview--limits).
+
+## Feature Requests/Discussions
+
+If you notice an issue or have a feature request (existing or new endpoint), we invite you to open an issue on [Koios Artifacts](https://github.com/cardano-community/koios-artifacts) repository first, and then feel free to discuss the topic in [Koios discussions](https://t.me/+zE4Lce_QUepiY2U1) group.
+
+## Participating as instance provider
+
+There will always be an audience who might want to run everything locally instead of relying on any external connection. Typically, these would be heavy users of the API (explorers/wallet providers/marketplaces), who'd like to override default API limits, have customisation requirements, remove latency hops OR enthusiasts who'd like to contribute back and add to the stregth of the API layer.
+
+For this audience, we've tried to leverage existing widely adopted `guild-operators` suite to build simpler scripts that'd allow you to create a gRest instance. This independent gRest instance will give you 100% of features and compatibility with the API documentation hosted on api.koios.rest (assuming it's built from latest `koios` tags).
+
+### Infrastructure Sizing
+
+Most of the infrastructure sizing at the moment is dependent on sizing of `cardano-node`, `cardano-db-sync` and `postgres`, and their consumption is different depending on network you connect to. You can find the official `cardano-db-sync` documentation for system requirements [here](https://github.com/input-output-hk/cardano-db-sync#system-requirements). But as a basic recommendation, if you're running a typical dbsync+node+postgres instance, we would recommend you to go for 64GB RAM for Mainnet, 16GB for Testnet and 8GB for Guildnet.
```

Comment:
Perhaps it should be stated here that this information refers to the status in May 2022.